plugins/pulp_rpm/plugins/profilers/yum.py

```diff
@@ -403,7 +428,7 @@ def _translate_erratum(unit, repo_ids, consumer, conduit):
         # Get rpm dicts from errata
         errata_rpms = YumProfiler._get_rpms_from_errata(errata)
         _logger.info(
-            "Errata <%s> refers to %s updated rpms of: %s" % (errata.unit_key['id'],
+            "Errata <%s> refers to %s updated rpms of: %s" % (errata.unit_key['errata_id'],
```

Comment:
This was a katello blocker over the weekend, fixed here: https://github.com/pulp/pulp_rpm/pull/786

If this one is merged monday, we can close that. If so, you should update the associated bug https://pulp.plan.io/issues/1604